main/parse.c

```diff
@@ -54,6 +55,8 @@ typedef struct {
 	enum specType specType;
 }  parserCandidate;
 
+static ptrArray *parsersUsedInCurrentInput;
```

Comment:
> This may be about style but in my experience marking static is enough.

yes a static variable has defined initialization rules, and here it's equivalent ton setting it to `NULL`  my remark is purely style, to make clear you expect this default value.

> However, as far as I can remember, I worked on eliminating UNreachable leak.

I remember you removing this kind of static variables holding "cache" value.  But anyway, it's no biggie, and if the main part has many it won't change too much, so it's fine.